static/js/components/Header.js

```diff
@@ -1,24 +1,28 @@
 import React from 'react';
 import LoginButton from '../containers/LoginButton';
-import Link from 'react-router/lib/Link';
+import { Navbar, NavItem } from 'react-bootstrap';
 
 class Header extends React.Component {
-  render() {
-    return <div id="header">
-      <div className="header-logo">
-        <a href="/">
-          <img src="/static/images/mit-micromasters.png" alt="MIT Micromasters"/>
-        </a>
-      </div>
-      <div className="header-links-dashboard">
-        <Link to="/dashboard" activeClassName="selected">
-          Dashboard
-        </Link>
-      </div>
-      <div className="header-links-login">
-        <LoginButton />
-      </div>
-    </div>;
+  render () {
+    return (
+      <Navbar bsStyle="default" fluid={true}>
+        <Navbar.Header>
+          <Navbar.Brand>
+            <a href="/">
+              <img src="/static/images/logo-micromasters@2x.png" width="215" alt="MIT Micromasters" />
+            </a>
+          </Navbar.Brand>
+          <ul className="nav navbar-toolbar navbar-left">
+            <NavItem href="/dashboard">
```

Comment:
yes, I think so. I think a `Link` with the right `className` set will be styled correctly